lib/ar_state_machine/configuration.rb

```diff
@@ -0,0 +1,11 @@
+module ARStateMachine
+  class Configuration
+
+    attr_accessor :system_id, :should_log_state_change
+
+    def initialize
+      self.should_log_state_change = true
```

Comment:
would probably be because my tests don't have a database? getting `'default_env' database is not configured. Available: []`